### DISCUSSION TITLE
Counter on the Readme page

### DISCUSSION THREAD

**Discussion by lovepeacejoy404** [community/community#21583] [Other Feature Feedback, Questions, & Ideas]
<pre><code>Hi, I would like to know if it is possible to insert a counter or statistics   
on the Readme page of my Github project to find out how many people   
download or clone the project. Thank you
</code></pre>

**Comment by Wabri** [comment]
<p>You can find a lot of badges here: <a href="https://shields.io" rel="nofollow noopener">https://shields.io</a></p>
<p>For example, the badge for number of download is this:</p>
<p><img alt=""></p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/lovepeacejoy404">@lovepeacejoy404</a>,</p>
<p>Thank you for being here! Another helpful tool could be to check out your traffic analytics in GitHub.</p>
<p>GitHub currently only offers the Traffic Graph for this data:</p>
<p><a href="https://help.github.com/articles/viewing-traffic-to-a-repository/" rel="nofollow noopener">https://help.github.com/articles/viewing-traffic-to-a-repository/</a></p>
<p>I hope this helps!</p>
<p>Andrea</p>

**Comment by khanfarhan10** [comment]
<p>Try using <a href="https://hitcounter.pythonanywhere.com/" rel="noopener nofollow ugc">https://hitcounter.pythonanywhere.com/</a></p>

**Comment by gayanvoice** [comment]
<p>There is a GitHub Action that uses GitHub API. It updates daily and does not require any servers or databases. It automatically saves changes into the repository. Go to the template <a href="https://github.com/gayanvoice/github-insights-template" rel="noopener nofollow ugc">GitHub Insights Template</a></p>

**Comment by antonkomarev** [comment]
@lovepeacejoy404 you can use [free service Ÿ HŸPE](https://yhype.me/) to collect info about your GitHub profile visits, extended repository and organization statistics: https://yhype.me/

Or use [free service GitHub Profile Views Counter](https://github.com/antonkomarev/github-profile-views-counter) if you only need a basic counter on profile page: https://github.com/antonkomarev/github-profile-views-counter
